.github/workflows/maintenance.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: '*/5 * * * *'
```

Comment:
It seems aggressive to run every 5 minutes. I thought it was enough to run it daily. What do you think?

Reply:
usually unsubscribe from notifications from the generated PRs